code/modules/client/client procs.dm

```diff
@@ -412,7 +412,10 @@
 	// Most assets are now handled through global_cache.dm
 	getFiles(
 		'html/search.js', // Used in various non-NanoUI HTML windows for search functionality
-		'html/panels.css' // Used for styling certain panels, such as in the new player panel
+		'html/panels.css', // Used for styling certain panels, such as in the new player panel
+		'fonts/crystal_bearers.ttf', // grey accent font
+		'fonts/DrippyWatercolor.ttf', //slime accent font
```

Comment:
The actual sending of the fonts could be called in send_resources- just think that it shouldn't be directly in the central "getFiles" proc, since, 1, it means that adding more fonts will quickly start to make a mess in this proc, and 2, depending on how getFiles() works, it could delay the search and panels (which are mostly used in admin tools) from working.